src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlConvertTranslator.cs

```diff
@@ -16,6 +16,7 @@ public class NpgsqlConvertTranslator : IMethodCallTranslator
         {
             [nameof(Convert.ToBoolean)] = "bool",
             [nameof(Convert.ToByte)]    = "smallint",
+            [nameof(Convert.ToDateTime)]    = "timestamp with time zone",
```

Comment:
nit: can you please reindent the other lines so they align as before?

Reply:
Done.